sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -509,6 +509,25 @@ func (m *networkFlowManager) enrichConnection(conn *connection, status *connStat
 		lookupResults = m.clusterEntities.LookupByEndpoint(conn.remote)
 	}
 
+	var port uint16
+	var direction string
+	if conn.incoming {
+		direction = "ingress"
+		port = conn.local.Port
+	} else {
+		direction = "egress"
+		port = conn.remote.IPAndPort.Port
+	}
+
+	metricDirection := prometheus.Labels{
+		"direction": direction,
+		"namespace": container.Namespace,
+	}
+
+	defer func() {
+		status.used = true
+	}()
```

Comment:
There were some paths in the code that ended without setting `used = true` (see `if isFresh`). Do you think we can set them as `used` without consequences?

I am myself not fully sure here, I think it should be fine.

Reply:
You're right actually, this was making some tests fail. I changed the cases in which status is set to true to match the original code.